templates/vue/src/components/Lightbox/media/VideoMedia/YouTubeMedia.vue

```diff
@@ -254,7 +254,6 @@ export default {
 
 <style lang="scss" scoped>
 .embed-responsive {
-  max-height: calc(100vh - 120px);
```

Comment:
Does this need to be removed? Same as the other change above... In the PR description it says that it was the help modal causing issues so I'm wondering if these changes are also part of the issue and that this fixes those issues without causing new problems.

Reply:
The line was removed to avoid a possible second bug, where if the browser height is decreased, the iframe is also resized vertically: 
![image](https://user-images.githubusercontent.com/11191061/130288695-2434eae7-2463-485b-8147-0a6938e6f44f.png)
If this is intended behaviour, scoping the CSS rule seem to be the only thing necessary to fix the video player controls being cut off bug from my testing.

Reply:
Yes, this is intended. I will remove this code then.